app/assets/javascripts/discourse/app/components/user-fields/confirm.hbs

```diff
@@ -1,22 +1,36 @@
-{{#if this.field.name}}
-  <label class="control-label">
-    {{this.field.name}}
-    {{~#unless this.field.required}} {{i18n "user_fields.optional"}}{{/unless~}}
-  </label>
-{{/if}}
+<PluginOutlet
+  @name="confirm-title-wrapper"
```

Comment:
This name needs to be a lot more specific. It needs to be clear that this is related to user fields